pkg/apis/crd/v1beta1/types.go

```diff
@@ -262,6 +263,95 @@ type IPPoolUsage struct {
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// IPPool defines one or multiple IP sets that can be used for flexible IPAM feature. For instance, the IPs can be
+// allocated to Pods according to IP pool specified in Deployment annotation.
+type IPPool struct {
```

Comment:
don't we need a conversion webhook for backwards-compatibility?

For reference, we currently have one for the `ClusterGroup` CRD:
- https://github.com/antrea-io/antrea/blob/88f2193e063eab4924b9b7419d20c5f65875f550/pkg/controller/networkpolicy/convert.go#L32
- https://github.com/antrea-io/antrea/blob/88f2193e063eab4924b9b7419d20c5f65875f550/pkg/apiserver/apiserver.go#L318

Reply:
Thanks for the information. I will check this part and update.

Reply:
Done